[2025-02-07T10:16:13.314+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-07T10:16:13.325+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: cluster_deployment.run_pre_check manual__2025-02-07T10:15:43.204776+00:00 [queued]>
[2025-02-07T10:16:13.330+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: cluster_deployment.run_pre_check manual__2025-02-07T10:15:43.204776+00:00 [queued]>
[2025-02-07T10:16:13.330+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2025-02-07T10:16:13.336+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonVirtualenvOperator): run_pre_check> on 2025-02-07 10:15:43.204776+00:00
[2025-02-07T10:16:13.341+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=217) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-07T10:16:13.342+0000] {standard_task_runner.py:72} INFO - Started process 218 to run task
[2025-02-07T10:16:13.342+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'cluster_deployment', 'run_pre_check', 'manual__2025-02-07T10:15:43.204776+00:00', '--job-id', '5', '--raw', '--subdir', 'DAGS_FOLDER/cluster_deployment.py', '--cfg-path', '/tmp/tmpbgyoeodh']
[2025-02-07T10:16:13.343+0000] {standard_task_runner.py:105} INFO - Job 5: Subtask run_pre_check
[2025-02-07T10:16:13.388+0000] {task_command.py:467} INFO - Running <TaskInstance: cluster_deployment.run_pre_check manual__2025-02-07T10:15:43.204776+00:00 [running]> on host ***-worker-{{.Task.Slot}}
[2025-02-07T10:16:13.444+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='cluster_deployment' AIRFLOW_CTX_TASK_ID='run_pre_check' AIRFLOW_CTX_EXECUTION_DATE='2025-02-07T10:15:43.204776+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-07T10:15:43.204776+00:00'
[2025-02-07T10:16:13.445+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-02-07T10:16:13.445+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-02-07T10:16:13.446+0000] {logging_mixin.py:190} INFO - Current task name:run_pre_check state:running start_date:2025-02-07 10:16:13.325433+00:00
[2025-02-07T10:16:13.446+0000] {logging_mixin.py:190} INFO - Dag name:cluster_deployment and current dag run status:running
[2025-02-07T10:16:13.447+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-02-07T10:16:13.447+0000] {baseoperator.py:421} WARNING - PythonVirtualenvOperator.execute cannot be called outside TaskInstance!
[2025-02-07T10:16:13.448+0000] {process_utils.py:186} INFO - Executing cmd: /home/***/.local/bin/python -m virtualenv /tmp/venvxd31axys --system-site-packages --python=python
[2025-02-07T10:16:13.449+0000] {process_utils.py:190} INFO - Output:
[2025-02-07T10:16:13.946+0000] {process_utils.py:194} INFO - created virtual environment CPython3.12.8.final.0-64 in 132ms
[2025-02-07T10:16:13.947+0000] {process_utils.py:194} INFO -   creator CPython3Posix(dest=/tmp/venvxd31axys, clear=False, no_vcs_ignore=False, global=True)
[2025-02-07T10:16:13.948+0000] {process_utils.py:194} INFO -   seeder FromAppData(download=False, pip=bundle, via=copy, app_data_dir=/home/***/.local/share/virtualenv)
[2025-02-07T10:16:13.948+0000] {process_utils.py:194} INFO -     added seed packages: pip==24.3.1
[2025-02-07T10:16:13.948+0000] {process_utils.py:194} INFO -   activators BashActivator,CShellActivator,FishActivator,NushellActivator,PowerShellActivator,PythonActivator
[2025-02-07T10:16:13.963+0000] {process_utils.py:186} INFO - Executing cmd: /tmp/venvxd31axys/bin/pip install -r /tmp/venvxd31axys/requirements.txt
[2025-02-07T10:16:13.965+0000] {process_utils.py:190} INFO - Output:
[2025-02-07T10:16:14.859+0000] {process_utils.py:194} INFO - Requirement already satisfied: pandas==2.1.4 in /home/***/.local/lib/python3.12/site-packages (from -r /tmp/venvxd31axys/requirements.txt (line 1)) (2.1.4)
[2025-02-07T10:16:14.864+0000] {process_utils.py:194} INFO - Requirement already satisfied: numpy<2,>=1.26.0 in /home/***/.local/lib/python3.12/site-packages (from pandas==2.1.4->-r /tmp/venvxd31axys/requirements.txt (line 1)) (1.26.4)
[2025-02-07T10:16:14.864+0000] {process_utils.py:194} INFO - Requirement already satisfied: python-dateutil>=2.8.2 in /home/***/.local/lib/python3.12/site-packages (from pandas==2.1.4->-r /tmp/venvxd31axys/requirements.txt (line 1)) (2.9.0.post0)
[2025-02-07T10:16:14.865+0000] {process_utils.py:194} INFO - Requirement already satisfied: pytz>=2020.1 in /home/***/.local/lib/python3.12/site-packages (from pandas==2.1.4->-r /tmp/venvxd31axys/requirements.txt (line 1)) (2024.2)
[2025-02-07T10:16:14.865+0000] {process_utils.py:194} INFO - Requirement already satisfied: tzdata>=2022.1 in /home/***/.local/lib/python3.12/site-packages (from pandas==2.1.4->-r /tmp/venvxd31axys/requirements.txt (line 1)) (2024.2)
[2025-02-07T10:16:14.868+0000] {process_utils.py:194} INFO - Requirement already satisfied: six>=1.5 in /home/***/.local/lib/python3.12/site-packages (from python-dateutil>=2.8.2->pandas==2.1.4->-r /tmp/venvxd31axys/requirements.txt (line 1)) (1.17.0)
[2025-02-07T10:16:15.241+0000] {process_utils.py:194} INFO - 
[2025-02-07T10:16:15.242+0000] {process_utils.py:194} INFO - [notice] A new release of pip is available: 24.3.1 -> 25.0
[2025-02-07T10:16:15.242+0000] {process_utils.py:194} INFO - [notice] To update, run: python -m pip install --upgrade pip
[2025-02-07T10:16:15.293+0000] {python.py:513} INFO - Use 'pickle' as serializer.
[2025-02-07T10:16:15.303+0000] {process_utils.py:186} INFO - Executing cmd: /tmp/venvxd31axys/bin/python /tmp/venv-call7bsftly3/script.py /tmp/venv-call7bsftly3/script.in /tmp/venv-call7bsftly3/script.out /tmp/venv-call7bsftly3/string_args.txt /tmp/venv-call7bsftly3/termination.log
[2025-02-07T10:16:15.304+0000] {process_utils.py:190} INFO - Output:
[2025-02-07T10:16:17.157+0000] {process_utils.py:194} INFO - [2025-02-07T10:16:17.156+0000] {script.py:23} INFO - Starting run pre-checks...
[2025-02-07T10:16:17.157+0000] {process_utils.py:194} INFO - [2025-02-07T10:16:17.156+0000] {script.py:26} INFO - Pre Deployment Report: {'services_status': {'running': 4}, 'deployment_ready': True, 'timestamp': '2025-02-07T10:15:52.756080'}
[2025-02-07T10:16:17.158+0000] {process_utils.py:194} INFO - [2025-02-07T10:16:17.157+0000] {script.py:27} INFO - Download Report: {'stacks': ['baseos', 'gi'], 'total_size_mb': 1200.0, 'total_time_seconds': 240, 'download_speed_mbs': 5.0, 'status': 'success', 'details': [{'stack': 'baseos', 'size_mb': 1000, 'status': 'completed', 'time': 180}, {'stack': 'gi', 'size_mb': 200, 'status': 'completed', 'time': 60}], 'timestamp': '2025-02-07T10:16:11.327069'}
[2025-02-07T10:16:17.158+0000] {process_utils.py:194} INFO - [2025-02-07T10:16:17.158+0000] {script.py:42} INFO - Pre-checks completed:
[2025-02-07T10:16:17.159+0000] {process_utils.py:194} INFO - Total memory usage: 1536%
[2025-02-07T10:16:17.159+0000] {process_utils.py:194} INFO - Status: PASS
[2025-02-07T10:16:17.497+0000] {python.py:240} INFO - Done. Returned value was: {'total_memory_usage_mb': 1536, 'checks_passed': True, 'timestamp': '2025-02-07T10:16:17.158311'}
[2025-02-07T10:16:17.519+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-02-07T10:16:17.520+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=cluster_deployment, task_id=run_pre_check, run_id=manual__2025-02-07T10:15:43.204776+00:00, execution_date=20250207T101543, start_date=20250207T101613, end_date=20250207T101617
[2025-02-07T10:16:17.527+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-02-07T10:16:17.528+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-02-07T10:16:17.528+0000] {logging_mixin.py:190} INFO - Dag name:cluster_deployment queued_at:2025-02-07 10:15:43.217670+00:00
[2025-02-07T10:16:17.528+0000] {logging_mixin.py:190} INFO - Task hostname:***-worker-{{.Task.Slot}} operator:PythonVirtualenvOperator
[2025-02-07T10:16:17.570+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-07T10:16:17.594+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-02-07T10:16:17.596+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
